utilities/cli/util.py

```diff
@@ -1353,9 +1354,41 @@ def collect_env_info() -> None:
     collect_python_info()
     collect_docker_info()
     collect_cuda_gpu_info()
+    collect_sccache_info()
     collect_environment_variables()
 
 
+def get_sccache_dir(env: Optional[dict[str, str]] = None) -> str:
+    source_env: dict[str, str] = env if env is not None else os.environ  # type: ignore[assignment]
+    return source_env.get("SCCACHE_DIR") or str(Path.home() / ".cache" / "sccache")
+
+
+def collect_sccache_info() -> None:
+    """Collect and display sccache-related information"""
+    print(f"\n{Color.blue('sccache Information:')}")
+
+    enable_val, enabled = get_env_bool("HOLOHUB_ENABLE_SCCACHE", default=False)
+    print(f"  HOLOHUB_ENABLE_SCCACHE: {enable_val} ({'enabled' if enabled else 'disabled'})")
+
+    sccache_bin = shutil.which("sccache")
+    version = run_info_command(["sccache", "--version"]) if sccache_bin else None
+    print(f"  sccache binary: {sccache_bin or '(not found in PATH)'}")
+    print(f"  sccache version: {version or '(unavailable)'}")
+
+    effective_dir = get_sccache_dir()
+    print(f"  Effective SCCACHE_DIR: {effective_dir}")
+
+    sccache_items = sorted(
+        (k, os.environ.get(k) or "(not set)") for k in os.environ if k.startswith("SCCACHE_")
+    )
+    if sccache_items:
+        print("  SCCACHE_* environment variables:")
+        for key, value in sccache_items:
+            print(f"    {key}: {value}")
```

Comment:
I do print this everytime we start building if sccache is enabled, since I feel like it's just as valid as the CMAKE_ variables that are printed after by cmake to understand the env. Do you think we should remove it from the build if we can see them here instead?